olive/systems/azureml/aml_system.py

```diff
@@ -278,12 +293,20 @@ def _create_step(
         parameters.append("--pipeline_output ${{outputs.pipeline_output}}")
 
         cmd_line = f"python {script_name} {' '.join(parameters)}"
+        resources = None
+        if vc_config:
+            resources = {
+                "instance_type": vc_config.instance_type,
+                "instance_count": instance_count,
+                "properties": {"AISuperComputer": {**vc_config.extra_args} if vc_config.extra_args else {}},
```

Comment:
Double check: is this `AISuperComputer` a fixed value always?

Reply:
Yes I think so.